### DISCUSSION TITLE
Welcome to blackhat-simulator Discussions!

### DISCUSSION THREAD

**Discussion by stautonico** [stautonico/blackhat-simulator#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jon-117** [comment]
Hey there! I saw your post on DEV about this after playing Grey Hack and started looking for a more realistic game like you were when you started. Do you plan to continue toward this or have you abandoned the project?

**Reply by stautonico** [reply]
I might continue this project in the future, but it may be difficult since I have other things to do. I started re-writing this in C++ at some point, but couldn't get it to a point where I was happy with it. Maybe sometime soon I will. Having other people be interested in it could spark my enthusiasm for the project again.
